src/screens/WalletConnectApprovalSheet.js

```diff
@@ -177,7 +177,7 @@ export default function WalletConnectApprovalSheet() {
   const timedOut = params?.timedOut;
   const failureExplainSheetVariant = params?.failureExplainSheetVariant;
   const chainIds = meta?.chainIds; // WC v2 supports multi-chain
-  const chainId = chainIds?.[0] || 1; // WC v1 only supports 1
+  const chainId = meta?.proposedChainId || chainIds?.[0] || 1; // WC v1 only supports 1
```

Comment:
feels weird to add another chainId value to the meta obj

Reply:
This is because we're reusing the wc prompt.  Mocking a WC request would be worse IMO